tests/specs/upgrade/out/upgrade.out

```diff
@@ -1,5 +1,4 @@
 Current Deno version: [WILDCARD]
-Downloading https://github.com/denoland/deno/releases/download/v1.43.2/deno-[WILDCARD].zip
```

Comment:
Why are these line gone now? I think we have a bigger problem here, that these `upgrade` tests depend on actually downloading binaries from a remote server. Any ideas how we could make them self-contained and not depend on remote server but instead testing server?

Reply:
I removed the line because we have the progress bar. Can add it back but feels too verbose?

Re local testing: we can probably have a env `DENO_DL_BASE_URL` that points to a file server with binaries.